rust/connlib/snownet/src/node.rs

```diff
@@ -1283,10 +1256,15 @@ struct Connection {
     // Socket addresses from which we might receive data (even before we are connected).
     possible_sockets: HashSet<SocketAddr>,
 
+    buffered_transmits: VecDeque<Transmit<'static>>,
+
     stun_servers: HashSet<SocketAddr>,
     turn_servers: HashSet<SocketAddr>,
 
     stats: ConnectionStats,
+
+    buffer: Box<[u8; MAX_UDP_SIZE]>,
```

Comment:
the `buffer`  name feels a bit generic here

Reply:
That is changing in the follow-up PR I linked above!